packages/ui/src/types/authMachine.ts

```diff
@@ -108,9 +108,11 @@ export type AuthInputNames = UserNameAlias | 'confirmation_code' | 'password';
 
 export type AuthInputAttributes = Record<AuthInputNames, InputAttributes>;
 
+export type AuthEventData = Record<PropertyKey, any>; // TODO: this should be typed further
```

Comment:
Now that we're adding facades, we should consider strongly typing this so that customers get vscode completions.